src/dispatchers/browserDispatcher.ts

```diff
@@ -45,21 +45,21 @@ export class BrowserDispatcher extends Dispatcher<Browser, channels.BrowserIniti
   }
 
   async crNewBrowserCDPSession(): Promise<channels.BrowserCrNewBrowserCDPSessionResult> {
-    if (this._object._options.name !== 'chromium')
+    if (this._object._options.name !== 'chromium' && this._object._options.name !== 'electron')
```

Comment:
maybe add a helper function like isChromiumBasedBrowser() since its duplicated logic which is "hard" to maintain.

Reply:
Yeah, something like isChromiumBased(). I wonder what happens with android? Can we do newCDPSession there? We probably should.

Reply:
Browsers now register themselves with `isChromium`. I changed the protocol so that `isChromium` is sent rather than `browserName`. And android now reports a ChromiumBrowserContext.